common/app/conf/switches/PerformanceSwitches.scala

```diff
@@ -66,6 +66,16 @@ trait PerformanceSwitches {
     exposeClientSide = false,
   )
 
+  val CircuitBreakerDcrSwitch = Switch(
+    SwitchGroup.Performance,
+    "circuit-breaker-dcr",
+    "If this switch is switched on then the DCR API circuit breaker will be operational",
+    owners = Seq(Owner.withEmail("dotcom.platform@theguardian.com")),
+    safeState = Off,
```

Comment:
Should the safeState be on i.e. the current behaviour?

Reply:
I asked the same thing in the original PR… and I’m tempted to agree with [Charlotte’s initial take](https://github.com/guardian/frontend/pull/26914#discussion_r1497309610).

Reply:
I’ve turned it on for the time being:

<img width="400" alt="image" src="https://github.com/guardian/frontend/assets/76776/d0161032-b16a-4a5d-b4a9-fb505754618b">